validator/validators/performance/performance_validator.go

```diff
@@ -131,6 +147,42 @@ func (s *PerformanceValidator) CalculateMetricStatsAndPackMetrics(metrics []type
 	return packIntoPerformanceInformation(uniqueID, receiver, dataType, fmt.Sprint(agentCollectionPeriod), commitHash, commitDate, map[string]interface{}{dataRate: performanceMetricResults}), nil
 }
 
+func (s *PerformanceValidator) CalculateWindowsMetricStatsAndPackMetrics(statistic []*cloudwatch.GetMetricStatisticsOutput) (PerformanceInformation, error) {
+	var (
+		receiver               = s.vConfig.GetPluginsConfig()[0] //Assuming one plugin at a time
```

Comment:
Could this array ever be empty? I presume no.

Reply:
`GetPluginsConfig()` will get the `receivers` variable from the `parameters.yml` file, so it is possbile for the array to be empty if there was a human error to not input any value for the variable. But realistically no, the array should never be empty.

Reply:
Considering these are integration tests and we would want to know that this is empty (panic) then this seems appropriate.